_config.yml

```diff
@@ -1,13 +1,38 @@
+address: 
+  - Address Line 1
+  - Address Line 2
+  - City, State, AZ 12345
```

Comment:
The address in the footer is actually getting removed from the standards, it hasn't been formally announced yet but its forthcoming. The reason for this is it has confused Google in the past as to where 'ASU' is. eg: [the honors college](https://barretthonors.asu.edu/) 

Personally I'm a little conflicted with this since our address in the footer gets quite a bit of use internally.